tests/dialects/test_starrocks.py

```diff
@@ -5,6 +5,33 @@
 class TestStarrocks(Validator):
     dialect = "starrocks"
 
+    def test_ddl(self):
+        ddl_sqls = [
+            "DISTRIBUTED BY HASH (col1) BUCKETS 1",
+            "DISTRIBUTED BY HASH (col1)",
+            "DISTRIBUTED BY RANDOM BUCKETS 1",
+            "DISTRIBUTED BY RANDOM",
+            "DISTRIBUTED BY HASH (col1) ORDER BY (col1)",
+            "DISTRIBUTED BY HASH (col1) PROPERTIES ('replication_num'='1')",
+            "PRIMARY KEY (col1) DISTRIBUTED BY HASH (col1)",
+            "DUPLICATE KEY (col1, col2) DISTRIBUTED BY HASH (col1)",
+        ]
+
+        for i, properties in enumerate(ddl_sqls):
+            tbl = f"foo{i}"
+            with self.subTest(f"Testing create scheme: {properties}"):
+                self.validate_identity(
+                    f"CREATE TABLE {tbl} (col1 BIGINT, col2 BIGINT) {properties}"
+                )
```

Comment:
The different `tbl` name here shouldn't make a difference in these tests right? We're interested only for the properties, so we can remove the enumeration

Reply:
Got it